godotproject/Player.gd

```diff
@@ -277,7 +279,7 @@ func do_movement(delta):
 		handle_normal_horizontal_movement(delta)
 		
 		# Adjust Y velocity based on gravity and jump
-		velocity.y += gravity * delta
+		velocity.y += gravity * delta * wind_speed.y
```

Comment:
Shouldn't this be something like 
```suggestion
		velocity.y += gravity * delta
		velocity += delta * wind_speed
```
I think we should look at changing later but not for here, since it works now